build.gradle

```diff
@@ -406,15 +417,10 @@ allprojects {
     }
 
     applySubtypingChecker = { ->
-      if (shouldRunAllCheckers) {
```

Comment:
Let's enable in local runs as well, so that one doesn't need to pass `CI=true` or `./gradlew -PrunAllCheckers` for tests